src/components/layout/ProductForm.tsx

```diff
@@ -0,0 +1,212 @@
+                                        categoryMapping.map((category) => (
+                                            <option key={category.value} value={category.value}>{category.label}</option>
+                                        ))
+                                    }
+                                </select>
+                            </div>
+                            <div className="product-form-line">
+                                <label>Si vous voulez écraser le prix calculé par la somme des ingrédients :</label>
+                                <input
+                                    className="product-form-input"
+                                    id="price"
+                                    name="price"
+                                    type="number"
+                                    placeholder="Prix"
+                                    onChange={handleChange}
+                                    value={values.price}
+                                />€
+                            </div>
+
+                            <FieldArray
+                                name="ingredients"
+                                render={ingredientsOfProduct => (
+                                    <div>
+                                        <span
+                                            className="product-form-line"> Ingredient(s) du produit :</span>
+                                        <div className="product-form-line-ingredient-list">
+                                            {values.ingredients && values.ingredients.length > 0 ? (
+                                                values.ingredients.map((ingredient, index) => (
+                                                    <div
+                                                        className="product-form-line-ingredient-list"
+                                                        key={index}>
+                                                        <span>{ingredient.name}</span>
+                                                        <button
+                                                            className="product-form-button-ingredient-list"
+                                                            type="button"
+                                                            onClick={() => ingredientsOfProduct.remove(index)}>
+                                                            X
+                                                    </button>
+                                                    </div>
+                                                ))
+                                            ) : (<div className="product-form-line">Aucun ingredients lié au produit</div>)}
+                                        </div>
+                                        <div className="product-form-line">
+                                            <label>Ajouter un ingrédient au produit :</label>
```

Comment:
Si tu mets des labels, bind les à des id avec `htmlFor`
```suggestion
<label htmlFor="ingredient-to-add">Ajouter un ingrédient au produit:</label>`
<select id="ingredient-to-add"
```